package.json

```diff
@@ -2505,8 +2505,6 @@
           },
           "description": "%cmake-tools.configuration.cmake.ctestDefaultArgs.description%",
           "default": [
-            "-T",
-            "test",
```

Comment:
Furthermore this also fixes the issue vscode warning about not finding DartConfiguration.tcl

Since not all projects `include(CTest)` for testing.